students/jemielity_kamil/lesson_05_lists/comma_code.py

```diff
@@ -0,0 +1,14 @@
+def comma_code(elements):
+    sentence = ''
+    length = len(elements)
+    for i in range(length):
+        if i is length - 1:
+            sentence += 'and ' + elements[i]
```

Comment:
For improved readability, it's better to write the code responsible for adding ', ' and then for ' and '

The output is incorrect, it should not includes a comma before ' and' statement:
```
apples, bananas, and tofu
```

My proposal:
```
def comma_code(elements):
    sentence = ''
    length = len(elements)
    for i, element in enumerate(elements):
        if i > 0:
            # Not the first element
            if i < length - 1:
                # Not the last element
                sentence += ', '
            else:
                # Last element
                sentence += ' and '
        sentence += element
    return sentence
```

Reply:
Sorry, I didn't notice it. You can leave the code as it is